docs/product-updates/2023-06-14-bump-cli-2-7-0.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Bump.sh CLI v2.7.0
+tags: [New]
+image: /files/changelog/bump-sh-cli-2-7-0.png
+---
+
+![bump-sh-cli-2-7-0.png](/files/changelog/bump-sh-cli-2-7-0.png)
+
+#### Multi-file Deployment
+
+Deploying multiple files on your Hub has never been this effortless. You can now provide a directory path to deploy all files within a directory to your Bump.sh hub.
+
+For a more detailed walkthrough on how this works, check out the feature in action in our [latest release notes](https://github.com/bump-sh/cli/releases/tag/v2.7.0).
+
+#### Live Preview
+
+Using the Live Preview feature  `bump preview --live my-definition-api.yml` now creates an immediate preview without waiting for the first change on the target file.
+
+Keeping up with the latest version of our CLI is now easier, thanks to a new warning that will alert you if you're running an outdated version.
```

Comment:
You should probably move this sentence in the “Other improvements” list :)

Reply:
I agree, moved ✅